packages/fluentui/accessibility/src/behaviors/Tree/treeAsListBehavior.ts

```diff
@@ -6,14 +6,18 @@ import { treeItemAsListItemBehavior } from './treeItemAsListItemBehavior';
 
 /**
  * @specification
- * Adds role 'list' to 'root' slot.
+ * Adds attribute 'role=listbox' to 'root' slot if 'selectable' property is true. Sets the attribute to 'list' otherwise.
+ * Adds attribute 'aria-multiselectable=true' to 'root' slot if 'selectable' property is true. Does not set the attribute otherwise.
  */
 export const treeAsListBehavior: Accessibility<TreeBehaviorProps> = props => {
   const behavior = treeBehavior(props);
   return _.merge(behavior, {
     attributes: {
       root: {
-        role: 'list',
+        role: props.selectable ? 'listbox' : 'list',
```

Comment:
why `listbox`?

Reply:
I was checking both:
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role
- https://www.w3.org/WAI/PF/aria/roles#listbox

Seems like this is the one to go for **list** that is **selectable**
Also in the original issue @kolaps33 made a [draft pull request](https://github.com/microsoft/fluentui/pull/15035/files) suggesting listbox

Reply:
Nice, I wonder if we should change something for Selectable Tree also then?!

Reply:
I'm not sure what do you mean? 
If you meant the normal tree behaviour, it's not list there, it's role='tree'.

Reply:
Sorry, my mistake, I wanted to say Selectable List

Reply:
Sorry for the late reply, somehow I missed this message. Selectable list is already using listbox+option role :)